screens/HomeScreen.js

```diff
@@ -212,7 +158,7 @@ const styles = StyleSheet.create({
     fontWeight: 'bold',
```

Comment:
Will this StyleSheet also be migrated into a separate style sheet for the homepage within the styles folder?

Reply:
Just read your PR comment – futures work!

Reply:
Today's (or tomorrow's) work ~!